computations/x1_16_chabauty.m

```diff
@@ -121,23 +142,30 @@ function EllipticChabauty(C, g, hk, HktoA)
 // failed cases
```

Comment:
We should remove this failed cases list. This was only something that was nice to have here while tweaking the code.